### DISCUSSION TITLE
Time Out when using NERv3 through spacy_llm although OPENAI KEY is valid

### DISCUSSION THREAD

**Discussion by fbalbachan** [explosion/spacy-llm#321] [General]
Time Out when using NERv3 through spacy_llm although OPENAI KEY is valid
This is happening in Pycharm but also in Google Colab. OPENAI KEY is valid

I tried with other API providers (Palm from Google ant it is fine)
I tried the OPENAI KEY and is valid

so, the problem is between Spacy and OPENAI
any idea what is happening?

**Comment by rmitsch** [comment]
Hi @fbalbachan, timeout errors usually occur when you're being rate-limited by OpenAI (i. e. you've been sending too many requests in a given time period) or when the OpenAI servers are too busy. This is likely not a problem between spaCy and OpenAI, but just on OpenAI's side.

Could you post the complete stack trace?

**Comment by innocent-charles** [comment]
is open ai recently down ?.. The paid api sucks or is just the spacy-llm latest version ?

**Reply by rmitsch** [reply]
Can you elaborate on which problems you are experiencing?
